ramls/circulation.raml

```diff
@@ -345,6 +345,174 @@ resourceTypes:
             body:
               text/plain:
                 example: "Internal server error"
+    /overdue-fine-policy:
```

Comment:
If these are in the `circulation-rules` interface, they should be in a separate RAML file describing that interface.

Maybe that wasn't split out correctly when the interface was extracted previously. Please could you help fix that, during this pull request.

Reply:
Fixed